Maquetacion/scss/tech.scss

```diff
@@ -1,35 +1,43 @@
-.tech {
-  align-items: center;
-  background-color: $green;
-  border-radius: 5px;
-  box-shadow: -10px -10px $white, -10px -10px 0 1px $green;
+.front {
   display: inline-flex;
-  flex-direction: column;
-  height: 320px;
-  margin: 5px 150px;
-  text-align: center;
-  width: 220px;
+  flex-wrap: wrap;
+  margin: 0 125px;
```

Comment:
I'd recommend using `max-width` and `margin: 0 auto` so that the content is displayed according to the design provided when screens are a bit larger.